hack/olm-registry/olm-artifacts-template.yaml

```
@@ -1,41 +1,111 @@
 apiVersion: v1
 kind: Template
 metadata:
-  name: olm-artifacts-template
+  name: selectorsyncset-template
 
 parameters:
-- name: REGISTRY_IMG
-  required: true
-- name: CHANNEL
-  value: staging
-- name: IMAGE_TAG
-  value: latest
+  - name: REGISTRY_IMG
+    required: true
+  - name: CHANNEL
+    value: staging
+    required: true
+  - name: IMAGE_TAG
+    required: true
+  - name: REPO_NAME
+    value: managed-upgrade-operator
+    required: true
 
 objects:
-- apiVersion: operators.coreos.com/v1alpha1
-  kind: CatalogSource
-  metadata:
-    name: managed-upgrade-operator-catalog
-  spec:
-    sourceType: grpc
-    image: ${REGISTRY_IMG}:${CHANNEL}-${IMAGE_TAG}
-    displayName: managed-upgrade-operator Registry
-    publisher: SRE
-
-- apiVersion: operators.coreos.com/v1alpha2
-  kind: OperatorGroup
-  metadata:
-    name: managed-upgrade-operator-og
-  spec:
-    targetNamespaces:
-    - managed-upgrade-operator
-
-- apiVersion: operators.coreos.com/v1alpha1
-  kind: Subscription
-  metadata:
-    name: managed-upgrade-operator
-  spec:
-    channel: ${CHANNEL}
-    name: managed-upgrade-operator
-    source: managed-upgrade-operator-catalog
-    sourceNamespace: managed-upgrade-operator
+  - apiVersion: hive.openshift.io/v1alpha1
+    kind: SelectorSyncSet
+    metadata:
+      labels:
+        managed.openshift.io/gitHash: ${IMAGE_TAG}
+        managed.openshift.io/gitRepoName: ${REPO_NAME}
+        managed.openshift.io/osd: "true"
+      name: managed-upgrade-operator
+    spec:
+      clusterDeploymentSelector:
+        matchLabels:
+          api.openshift.com/managed: "true"
+      resourceApplyMode: Sync
+      resources:
+        - apiVersion: v1
+          kind: Namespace
+          metadata:
+            name: managed-upgrade-operator
```

Change to `openshift-managed-upgrade-operator`
```suggestion
            name: openshift-managed-upgrade-operator
```